controller/src/main/java/org/jboss/as/controller/PrimitiveListAttributeDefinition.java

```diff
@@ -51,8 +51,9 @@
     }
 
 
-    public ModelType getValueType() {
-        return valueType;
+    @Override
+    public AttributeDefinition getValueType() {
```

Comment:
Why is the ValueType now returning an AttributeDefinition instead of a ModelType?

Reply:
because we need more information than just a type which an AttributeDefinition provides